AbletonMCP_Remote_Script/__init__.py

```
@@ -799,8 +1015,1196 @@ def _find_browser_item_by_uri(self, browser_or_item, uri, max_depth=10, current_
+                "clip_index": clip_index,
+                "looping": clip.looping
+            }
+        except Exception as e:
+            self.log_message("Error setting clip looping: " + str(e))
+            raise
+
+    def _set_clip_loop_points(self, track_index, clip_index, loop_start, loop_end):
+        """Set the loop start and end points of a clip"""
+        try:
+            if track_index < 0 or track_index >= len(self._song.tracks):
+                raise IndexError("Track index out of range")
```

Ruff (0.14.14)

[warning] Abstract `raise` to an inner function

(TRY301)